pygments_ansi_color/__init__.py

```diff
@@ -160,23 +170,39 @@ class AnsiColorLexer(pygments.lexer.RegexLexer):
     aliases = ('ansi-color', 'ansi', 'ansi-terminal')
     flags = re.DOTALL | re.MULTILINE
 
-    def __init__(self, *args, **kwargs):
+    bold: bool
+    fant: bool
+    fg_color: typing.Optional[str]
+    bg_color: typing.Optional[str]
+
+    def __init__(
+        self,
+        *args: typing.Iterable[typing.Any],
+        **kwargs: typing.Dict[typing.Any, typing.Any],
```

Comment:
this is wrong -- this would mean "each argument is an Iterable[Any]" and "each named argument is a Dict[Any, Any]" -- you want `*args: Any, **kwargs: Any`